rrmngmnt/storage.py

```diff
@@ -0,0 +1,127 @@
+import logging
+import shlex
+from rrmngmnt.service import Service
+
+logger = logging.getLogger(__name__)
+
+LV_CHANGE_CMD = 'lvchange -a %s %s/%s'
+
+
+class NFSService(Service):
+    """
+    Storage management class to maintain NFS services
+    """
+
+    def mount(self, source, target=None, opts=None):
+        """
+        Mounts source to target mount point
+
+        __author__ = "ratamir"
+        :param source: Full path to source
+        :type source: str
+        :param target: Path to target directory, if it is omitted then
+        temporary folder is created and it is the responsible of the
+        caller to remove it at the end
+        :type target
+        :param opts: List of options like: ['-t', 'nfs', '-o', 'vers=3']
+        :type opts: list
+        :return: Path to mount point if succeeded, None otherwise
+        :rtype: str
+        """
+        target = '/tmp/mnt_point' if target is None else target
+        cmd = ['mkdir', target]
+        self.logger.info(
+            "Creating directory %s to use as a mount point", target
+        )
+        rc, out, err = self.host.run_command(cmd)
+        if rc:
+            self.logger.error(
+                "Failed to create directory as a mount point for. "
```

Comment:
The sentence is incomplete: Failed to create a directory to be used as a mount point for %s (then pass source, or alternatively the target you tried to create with the appropriate text change)